<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFont</class>
 <widget class="QDialog" name="SelectFont">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Font</string>
  </property>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentFont">
    <font>
     <family>DejaVu Sans Mono</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fontColorPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="boldCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="italicCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sizeSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>12</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectFont</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SelectFont</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
